### PR TITLE
Expand yast2_bootloader

### PR DESCRIPTION
**Pull request by Zaoliang**
Please see my reference test at:
http://e13.suse.de/tests/2853

### CODE REVIEW

File: tests/yast2_gui/yast2_bootloader.pm [hunk 1]
```diff
@@ -8,11 +8,12 @@
...
 # without any warranty.
 
-# G-Summary: Add YaST2 UI tests
+# G-Summary: YaST2 UI test bootloader checks boot code option, kernel parameters,
+#	graphical console, bootloader option, foreign OS and protect boot loader
 #    Make sure those yast2 modules can opened properly. We can add more
 #    feature test against each module later, it is ensure it will not crashed
 #    while launching atm.
-# G-Maintainer: Max Lin <mlin@suse.com>
+# G-Maintainer: Zaoliang Luo <zluo@suse.com>
```
Comment (Soulofdestiny) [inline tests/yast2_gui/yast2_bootloader.pm]: please change 'G-Maintainer' to Maintainer

File: tests/yast2_gui/yast2_bootloader.pm [hunk 2]
```diff
@@ -8,11 +8,12 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# G-Summary: Add YaST2 UI tests
+# G-Summary: YaST2 UI test bootloader checks boot code option, kernel parameters,
```
Comment (Soulofdestiny) [inline tests/yast2_gui/yast2_bootloader.pm]: s/G-Summary/summary please

**Comment by Zaoliang** [comment]
okay, changed now

**Comment by Soulofdestiny** [comment]
Thanks, will merge after checks completed

**Pull request closed without merging**